test/jdk/java/security/SignedJar/spi-calendar-provider/TestSPISigned.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Red Hat, Inc.
+ * Copyright (c) 2022, 2025, Red Hat, Inc.
```

Comment:
Restore original line and insert an additional one.

```suggestion
 * Copyright (c) 2022, Red Hat, Inc.
 * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```